app/models/financial_transaction.rb

```diff
@@ -6,10 +6,11 @@ class FinancialTransaction < ActiveRecord::Base
   belongs_to :rental
   belongs_to :transactable, polymorphic: true
 
-  validates :adjustment, :rental_id, :initial_amount, presence: true
-  validates :initial_amount, numericality: { greater_than: 0 }
+  validates :rental_id, :amount, presence: true
+  validates :amount, numericality: { greater_than: 0 }
 
-  alias_attribute :base_amount, :initial_amount
+  alias_attribute :base_amount, :amount
```

Comment:
updated and i think you will be pleased

before 
![image](https://cloud.githubusercontent.com/assets/2114323/25455589/5f3fc2d8-2a9e-11e7-9791-61787e3df49e.png)

after
![image](https://cloud.githubusercontent.com/assets/2114323/25456200/3019db54-2aa0-11e7-9cc2-1b17d204fd7b.png)